media-sound/whipper/whipper-0.7.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python CD-DA ripper preferring accuracy over speed (forked from morituri)"
+HOMEPAGE="https://github.com/JoeLametta/${PN}"
+SRC_URI="https://github.com/JoeLametta/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	${PYTHON_DEPS}
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	>=dev-libs/libcdio-paranoia-0.94_p2
+	app-cdr/cdrdao
+	dev-python/pygobject:3=[${PYTHON_USEDEP}]
+	dev-python/python-musicbrainz-ngs[${PYTHON_USEDEP}]
+	media-libs/mutagen[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	dev-python/pycdio[${PYTHON_USEDEP}]
+	media-libs/libsndfile
+	media-libs/flac
+	media-sound/sox[flac]
+	dev-python/cddb-py[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+
+PATCHES=( "${FILESDIR}/${P}-cdparanoia-name-fix.patch" )
+
+src_prepare() {
+	sed -e 's|PREFIX = /usr/local|PREFIX = ${EPREFIX}/usr|' -i "${S}"/src/config.mk || die
```

Comment:
Can't you pass it as arg to `emake`? It generally sucks to sed paths in, since it's going to explode when `${EPREFIX}` contains something sed doesn't like.